Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/goblin_outerclothing.yml

```
@@ -0,0 +1,45 @@
+- type: entity
+  parent: ClothingOuterStorageToggleableBase
+  id: ClothingOuterCoatRobesGoblin
+  name: goblin robes
+  description: Traditional robes of goblinkind. Very regal.
+  components:
+  - type: Tag
+    tags:
+    - ClothingRobesGoblin
+  - type: Sprite
+    sprite: _NF/Clothing/OuterClothing/Coats/goblin_robes.rsi
+  - type: Clothing
+    sprite: _NF/Clothing/OuterClothing/Coats/goblin_robes.rsi
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadHatHoodGoblin
+    slot: head
+  - type: Construction
+    graph: ClothingOuterCoatRobesGoblinGraph
+    node: ClothingOuterCoatRobesGoblinNode
+
+- type: entity
+  parent: ClothingOuterEVASuitBase
+  id: ClothingOuterEVASuitGoblin
+  name: goblin EVA suit
+  description: Traditional robes of goblinkind turned in EVA.
```

"turned into EVA" is more acceptable, but I think this is better:
```suggestion
  description: Traditional robes of goblinkind made EVA-capable. Very regal.
```